MoviesApp/Model/MoviesListProvider.swift

```diff
@@ -50,26 +50,40 @@ class MoviesListManager {
         NotificationCenter.default.post(name: .ImageLoaded, object: movie)
     }
     
+    func setDetails(for movie: Movie, details: Details){
+        let index = getIndexOfSortedMovie(movie)
+        allMovies[index].details = details
+    }
+    
+    func setBackDrop(for movie: Movie, image: UIImage) {
```

Comment:
how about if we provide just the movie id and the details here? we can get the movie here by using getMovieById(:_)